ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-custom-control-plane-policy-edge-1.cfg

```diff
@@ -215,14 +215,6 @@ route-map RM-BGP-UNDERLAY-PEERS-IN permit 40
    description Mark prefixes originated from the LAN
    set extcommunity soo 192.168.42.1:511 additive
 !
-route-map RM-BGP-UNDERLAY-PEERS-OUT permit 10
-   description Advertise local routes towards LAN
-   match extcommunity ECL-EVPN-SOO
```

Comment:
add limitation regading route from eBGP peer in VRF default not advertised to the WAN - TODO